app/models/invitation.rb

```diff
@@ -0,0 +1,37 @@
+class Invitation < ActiveRecord::Base
+  include AASM
+
+  belongs_to :task, inverse_of: :invitations
+  has_one :paper, through: :task
+  belongs_to :invitee, class_name: "User", inverse_of: :invitations
+  belongs_to :actor, class_name: "User", inverse_of: :invitations
+
+  scope :pending, -> { where(state: "pending") }
```

Comment:
Ah, you are correct. This was put in before we included `aasm`. Thanks for the :eyes: